### PR TITLE
Jwt userdata

### PR DESCRIPTION
**Pull request by bburtin**

### CODE REVIEW

File: brkt_cli/aws/encrypt_ami_args.py [hunk 1]
```diff
@@ -148,3 +148,16 @@ def setup_encrypt_ami_args(parser):
         help=argparse.SUPPRESS,
         dest='key_name'
     )
+
+    # This option is still in development.
+    """
+    help=(
+        'JSON Web Token that the encrypted instance will use to '
```
Comment (bonzotronix) [inline brkt_cli/aws/encrypt_ami_args.py]: "encryptor instance"?

File: brkt_cli/aws/update_encrypted_ami_args.py [hunk 2]
```diff
@@ -146,3 +146,16 @@ def setup_update_encrypted_ami(parser):
         help=argparse.SUPPRESS,
         dest='encryptor_ami'
     )
+
+    # This option is still in development.
+    """
+    help=(
+        'JSON Web Token that the encrypted instance will use to '
```
Comment (bonzotronix) [inline brkt_cli/aws/update_encrypted_ami_args.py]: "encryptor instance"?

File: brkt_cli/aws/encrypt_ami_args.py (hunk 1 above)
Comment (bburtin) [inline brkt_cli/aws/encrypt_ami_args.py]: Probably bad terminology on my part.  It's not the encryptor instance.  It's the metavisor running the customer instance.  Maybe I should just say "metavisor."  The option is still hidden.  I'll fix the wording in a subsequent commit.

Comment (bonzotronix) [inline brkt_cli/aws/encrypt_ami_args.py]: Sorry, my misunderstanding about the purpose of these JWTs.  Grant set me straight.  Your verbiage is correct.

**Comment by bonzotronix** [comment]
LGTM, seen mostly from a 'userdata handling' perspective.  Maybe another reviewer with more JWT awareness should give a +2.

**Comment by gcallaghan** [comment]
LGTM

**Pull request merged**